packages/grafana-data/src/types/datasource.ts

```diff
@@ -281,6 +281,7 @@ export interface ExploreQueryFieldProps<
   TOptions extends DataSourceJsonData = DataSourceJsonData
 > extends QueryEditorProps<DSType, TQuery, TOptions> {
   history: any[];
+  onBlur?: () => void;
```

Comment:
maybe a bool option that describes what setting this does is more clear? 

runQueryOnBlur: boolean

Reply:
and would add that to the main QueryEditorProps interface.